docs/source/guides/custom_types_and_type_inference.ipynb

```diff
@@ -14,7 +14,7 @@
     "\n",
     "## Viewing Built-In Logical Types\n",
     "\n",
-    "To view all of the default LogicalTypes in Woodwork, use the `list_logical_types` function. If the existing types are not sufficient for your needs, you can create and register new LogicalTypes for use in creating DataTables and DataColumns."
+    "To view all of the default LogicalTypes in Woodwork, use the `list_logical_types` function. If the existing types are not sufficient for your needs, you can create and register new LogicalTypes for use with Woodwork initialized DataFrames and Series."
```

Comment:
I think it might be useful to link to the tags and types guide (or insert that three-bullet summary of physical types, logical types, and semantic tags that I think Angela has said she references all the time) right at the beginning of this guide in case a user needs a reference to understand some of the Woodwork-specific language.

On the other hand, it might be safe to assume that anyone trying to customize Woodwork typing is fairly familiar with the language, so maybe it's not necessary.

Reply:
Yeah, I certainly view this as more of an advanced user guide, but it's easy enough to add a link to the types and tags guide if someone needs a refresher.

Reply:
Added a link